src/Controls/src/Core/Handlers/Items2/iOS/TemplatedCell2.cs

```diff
@@ -212,6 +212,10 @@ void BindVirtualView(View virtualView, object bindingContext, ItemsView itemsVie
 			if (PlatformHandler?.VirtualView is View view)
 			{
 				view.SetValueFromRenderer(BindableObject.BindingContextProperty, bindingContext);
+				if (view.Parent is null)
+				{
+					itemsView.AddLogicalChild(view);
```

Comment:
Are we not setting the Parent already before?
https://github.com/dotnet/maui/blob/7ae256363b4a3ba3df9706b249524c7277e82aa4/src/Controls/src/Core/Handlers/Items2/iOS/TemplatedCell2.cs#L209